model/src/main/resources/application.yml

```diff
@@ -0,0 +1,3 @@
+spring:
```

Comment:
You don't need to have model module. Even if you wanted it would be as lib, but not as Spring component